sublime_info/SublimeInfo.py

```diff
@@ -2,8 +2,11 @@
 import os
 import re
 import subprocess
-from shutilwhich import which
-from errors import STNotResolvedError, STBadLocationError
+from sublime_info.errors import STNotResolvedError, STBadLocationError
```

Comment:
Can we add a from `__future__ import absolute_import`? I think this might break Python 2 =/